### DISCUSSION TITLE
send push notifications

### DISCUSSION THREAD

**Discussion by sibisandaniel86** [appcelerator/titanium_mobile#13604] [Q&A] [answered]
Is it possible to implement push notifications ?  the use of ti.cloud and ti.cloudpush as stated in the documentation i imagine its not longer an option.

**Comment by m1ga** [comment, marked as answer]
of course you can. I have a full tutorial at https://fromzerotoapp.com/ah-push-it-use-firebase-push-in-your-app/

**Reply by sibisandaniel86** [reply]
Super , thank you m1ga .
